scripts/migrate_airtable_to_postgres.ts

```diff
@@ -0,0 +1,66 @@
+import 'reflect-metadata';
+
+import getRepository from './../lib/db.js';
+
+import {
+    airtable_api_key,
+    airtable_base,
+    airtable_table,
+} from './../lib/secrets_wrapper.js';
+
+import Airtable from 'airtable';
+
+Airtable.configure({
+    endpointUrl: 'https://api.airtable.com',
+    apiKey: airtable_api_key,
+    apiVersion: Airtable.apiVersion,
+    noRetryIfRateLimited: Airtable.noRetryIfRateLimited,
+});
+
+const base = Airtable.base(airtable_base);
+const table = base.table(airtable_table);
+
+(async () => {
+    const repo = await getRepository();
+
+    table
+        .select
+        // {
+        // // Selecting the first 100 records in Grid view:
+        // maxRecords: 100,
+        // view: 'Grid view',
+        // }
+        ()
+        .eachPage(
+            function page(records, fetchNextPage) {
+                // This function (`page`) will get called for each page of records.
+
+                records.forEach(async function (record) {
+                    console.log('Retrieved', record.get('id'));
+
+                    await repo.save({
+                        // `typeorm` requires all fields to be here for some reason, even though they should be optional, so I am just setting them to `false` or `''`.
+                        text: 'message',
+                        approved: false,
+                        viewed: false,
+                        uid_salt: '',
+                        uid_hash: '',
+                        staging_ts: '',
+                        published_ts: '',
+                        ...record.fields,
+                    });
```

Comment:
Should just be able to do `await repo.save(record.fields as Confession)`

Reply:
ended with 

```
await repo.save({
  text: record.fields.text ?? 'message',
  approved: record.fields.approved ?? false,
  viewed: record.fields.viewed ?? false,
  uid_salt: record.fields.uid_salt ?? '',
  uid_hash: record.fields.uid_hash ?? '',
  staging_ts: record.fields.staging_ts ?? '',
  published_ts: record.fields.published_ts ?? '',,
  id: record.fields.id,
});
```
based on discussion on slack